filter_plugins/oo_filters.py

```diff
@@ -621,11 +621,23 @@ def to_padded_yaml(data, level=0, indent=2, **kw):
             return ""
 
         try:
-            transformed = yaml.safe_dump(data, indent=indent, allow_unicode=True, default_flow_style=False, **kw)
+            if isinstance(data, list):
+                _data = []
+                for d in data:
+                    _data.append(unicode(d))
+            elif isinstance(data, dict):
+                # Assumes all values are simple strings
+                _data = {}
+                for k, v in data.iteritems():
+                    _data[unicode(k)] = unicode(v)
+            else:
+                _data = data
+
```

Comment:
I think this needs to be done recursively. I tested with some options that get passed through the `to_padded_yaml` filter.

```ini
openshift_node_kubelet_args={'max-pods': ['40'], 'image-gc-high-threshold': ['90'], 'image-gc-low-threshold': ['80']}
osm_controller_args={'resource-quota-sync-period': ['10s']}
osm_api_server_args={'max-requests-inflight': ['400']}
```

Lists nested within dictionaries are added as strings to the master config.
```yaml
  apiServerArguments:
    max-requests-inflight: '[u''400'']'
  controllerArguments:
    resource-quota-sync-period: '[u''10s'']'
```